### ISSUE TITLE
Ubuntu 17.10 Pointer jumps 

### ISSUE THREAD

**Issue by rmasclef** [iberianpig/fusuma#68]
> TL;DR:
> - fusuma not launched with last linux kernel -> :white_check_mark: 
> - fusuma launched (with default ubuntu 17.10 kernel or higher) -> pointer jump 

Hello, 

I have an issue regarding the pointer behavior. Indeed, the pointer jumps around the screen when fusuma is launched.

The different triggers looks OK and I don't see any error message when launching fusuma in verbose mode.

I'm using **Ubuntu 17.10** with the last stable linux kernel `4.15.7-041507-generic)`.
The OS is installed on a Razer Blade 14" late 2017. 

The issue is known on that laptop **but** upgrading the linux kernel fixed it (when fusuma is not launched) so the problem seems to be related to fusuma :+1:

**Comment by rmasclef** [comment]
I'm gonna install an Ubuntu 16.04 LTS on that laptop and see if the bug is really related to fusuma or OS related.

**Comment by rmasclef** [comment]
A little update on this issue:

I'm often upgrading the linux kernel and I found some better behaviour lately (I don't launch fusuma automatically at startup, I launch it manually ...). Anyway, the problem is still there

FYI: I just installed Ubuntu 16;04 (yeah, finally). I did not installed fusuma yet, but, from now, I don't see any jumping cursor.

I'm gonna do some tests with fusuma to see what happens :+1:
